### PR TITLE
Sum value

### PR DESCRIPTION
**Pull request by andrewskerratt**
Added SumCards Value to Player class

### CODE REVIEW

**Comment by Jedi-Rogue1** [comment]
Andrew, your code looks clean and efficient. I cannot think of anything that I would recommend doing differently.

One thing I can think of that you might want to consider is writing a couple comments. Maybe like "Module adds up the cards in a player's hand", just to clarify what you are doing. I know we are in a small group with relatively simple tasks, but If you were in a large group with a large task, I think your teammates would appreciate it.

**Review by Jedi-Rogue1 — APPROVED** [review]

File: source/CardGames.py [hunk 1]
```diff
@@ -102,7 +102,13 @@ def addCard(self, card: Card, isKnown: bool = True):
 
   def clearHand(self):
     self.hand = []
-    self.knownCards = []
+    self.knownCards = [1]
```
Comment (JHerren726) [inline source/CardGames.py:105]: I'm not sure why you edited this part of the clearHand() function. It breaks the function.

File: source/CardGames.py [hunk 2]
```diff
@@ -102,7 +102,13 @@ def addCard(self, card: Card, isKnown: bool = True):
...
+
```
Comment (JHerren726) [inline source/CardGames.py:106]: The function looks great! Clean and concise.

I think we are all in agreement that we are going to use comments next sprint as well.

File: test/test_sample.py [hunk 3]
```diff
@@ -5,3 +5,4 @@ def test_answer():
     assert func(3) == 5
 
 print("hello world")
+print("yo")
```
Comment (JHerren726) [inline test/test_sample.py:8]: we were supposed to add our own test that tests the function we just added.

**Review by JHerren726 — APPROVED** [review]
Apparently I accidentally approved before I reviewed, but I left my comments!

**Review by TMUProfLee — COMMENTED** [review]
Next time make sure to add your test file to the pull request.

**Pull request merged**